endpoints/openrtb2/amp_auction.go

```diff
@@ -433,7 +435,7 @@ func (deps *endpointDeps) overrideWithParams(ampParams amp.Params, req *openrtb2
 		}
 	}
 
-	setAmpExt(req.Site, "1")
+	setAmpExtDirect(req.Site, "1")
```

Comment:
Not yet ready to use the request wrapper here. If you can believe it based on the size of this PR, I tried to limit the scope. :)